sig-cluster-lifecycle/grooming.md

```diff
@@ -0,0 +1,37 @@
+# SIG Cluster Lifecycle Grooming
+
+## Overview 
+
+The purpose of this document is to outline the current backlog grooming practices, or process, that are employed across several sub-projects under SIG Cluster Lifecycle.  By providing a semi-consistent set of best practices it allows developers to work across sub-projects fairly easily.  More importantly it signals direction and priority to developers who may be new to the sub-projects.  
+
+This document is an overview of the practices that are currently followed and any feedback, or refinement, is solicited. 
+
+## Process 
+
+### Planning a milestone
+For every sub-project under sig-cluster lifecycle, we highly recommend holding a planning session prior to beginning a new milestone.  The purpose of planning is to outline a set of high level deliverables that the community has decided that they want to work on for this upcomming milestone.  
```

Comment:
as the CI suggests `upcomming` -> `upcoming`

Reply:
Fixed.